docs/production-deployment/cloud/gcp-export-gcs.mdx

```diff
@@ -0,0 +1,168 @@
+3. After inputting the necessary values, click on **Verify**. 
+  You should be able to write to sink successfully. 
+  If not, please fix any errors or reach out to support for help.
+4. Clicking **Create** will complete the Export sink set up
+5. The page will auto-refresh and you should see the status “Enabled” on the Export screen. 
+  You are now ready to export workflow histories.
+6. You can toggle the enable button if you want to stop export and resume in the future. 
+  Note: when you re-enable the feature, it will start from the current point in time, and not from the time when you disable export.
+7. You can also delete export by clicking Delete.
+
+:::tip
+
+Don't forget to click Create at the end of your setup to confirm your export.
+
+:::
+
+### Using tcld
+
+To access export-related commands in tcld, please follow these steps:
+
+1. Download the latest version of tcld. [https://docs.temporal.io/cloud/tcld/#install-tcld]
+2. Make sure tcld version is v0.17.0 or above.
+3. Run the command: `tcld f toggle-export` and `tcld f toggle-gcp-sink`. 
+    This command enables the export GCP sink feature in tcld
+    1. After enabling the feature, you can find the export-related commands by running `tcld n export gcs`:
+    
+    ```bash
+    NAME:
+       tcld namespace export gcs - Manage GCS export sink
+    
+    USAGE:
+       tcld namespace export gcs command [command options] [arguments...]
+    
+    COMMANDS:
+       create, c    Create export sink
+       update, u    Update export sink
+       validate, v  Validate export sink
+       get, g       Get export sink
+       delete, d    Delete export sink
+       list, l      List export sinks
+       help, h      Shows a list of commands or help for one command
+    
+    OPTIONS:
+       --help, -h  show help
+    
+    ```
+    
+4. Run the `tcld n export gcs create` command and provide the following information:
+    - —namespace
+    - —sink-name
+    - —service-account-email
+    - —gcs-bucket
+    - For Example:
+    
+    ```bash
+    tcld n export gcs create -n test.ns --sink-name test-sink --service-account-email test-sink@test-export-sink.iam.gserviceaccount.com --gcs-bucket test-export-validation`
+    ```
+5. Check the status of this command by either viewing the Namespace Export status in the Temporal Cloud UI or using the following command and looking for the state of “Active”.
+
+```bash
+tcld n export gcs g -n test.ns --sink-name test-sink
+{
+	"name": "test.ns",
+	"resourceVersion": "b954de0c-c6ae-4dcc-90bd-3918b52c3f28",
+	"state": "Active",
+	"spec": {
+		"name": "test-sink",
+		"enabled": true,
+		"destinationType": "Gcs",
+		"s3Sink": null,
+		"gcsSink": {
+			"saId": "test-sink",
+			"bucketName": "test-export-validationn",
+			"gcpProjectId": "test-export-sink",
+		}
+	},
+	"health": "Ok",
+	"errorMessage": "",
+	"latestDataExportTime": "0001-01-01T00:00:00Z",
+	"lastHealthCheckTime": "2024-01-23T06:40:02Z"
+}
+```
+
+## **Monitoring after Export is enabled**
```

Comment:
Why is that header setup in bold ? IIRC it does not change anything in docusaurus ?

```suggestion
## Monitoring after Export is enabled
```